atlas_placement_hints/app/placement_hints.py

```diff
@@ -340,3 +340,59 @@ def thalamus(
         output_dir,
         has_hemispheres=True,
     )
+
+
+@app.command()
+@verbose_option
+@common_atlas_options
+@click.option(
+    "--metadata-path",
+    type=EXISTING_FILE_PATH,
+    required=False,
+    help=(
+        "(Optional) Path to the metadata json file. Defaults to "
+        f"`{str(METADATA_REL_PATH / 'cerebellum_metadata.json')}`"
```

Comment:
is the `str` here needed?

Reply:
ah no, I just copied another entry, I can remove them yep